src/app/features/home/home.module.ts

```diff
@@ -2,9 +2,20 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HomeRoutingModule } from './home-routing.module';
 import { HomeComponent } from './home.component';
+import { NavbarComponent } from './navbar/navbar.component';
+
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
 
 @NgModule({
-  declarations: [HomeComponent],
-  imports: [CommonModule, HomeRoutingModule],
+  declarations: [HomeComponent, NavbarComponent],
```

Comment:
for multiple declarations/imports, use multiline format like the next line

Reply:
The default configuration for "Prettier" the extension has a certain threshold for the long lines.
When it exceeds the threshold, it will split the imports to multiple lines.